test/built-ins/Temporal/Instant/prototype/toLocaleString/return-string.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2021 Kate Miháliková. All rights reserved.
```

Comment:
@FrankYFTang this is not your name ;)

Reply:
It is a strip down version from 
https://github.com/tc39/test262/blob/main/test/built-ins/Temporal/Instant/prototype/toLocaleString/options-conflict.js
so I have keep the credit to the author of that file